src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Remote/IRemoteSemanticTokensService.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Remote.Razor;
```

Comment:
I know you've made this consistent with other types in the same folder, but should all of them be renamed to `Microsoft.CodeAnalysis.Razor.Remote` to be consistent with the naming scheme with been using in Workspaces? (Yes, I know this is inconsistent with the `Microsoft.CodeAnalysis.Remote.Razor` project. 😔)

Reply:
I definitely just copied whatever was in the tag helpers service, since I had _so much_ trouble with service hub, but I think I _might_ just be brave enough to take on this sort of change now :)